Sources/Time/Documentation.docc/Common-Mistakes.md

```diff
@@ -0,0 +1,55 @@
+# Common Mistakes
+
+An overview of common mistakes made while dealing with **Time** or its concepts in general.
```

Comment:
Everywhere in the new markdown, I formatted the package's name to be bold, rather than `monospaced` to make it stand out more. That's just a personal style of _mine_, so please let me know if you have a different preference for referencing _modules_ or _libraries_

These docs are 98% verbatim from their source. I made links to Foundation docs where missing, as well as used DocC symbol linking to provide a richer experience.

The only formatting changes I made were to fit within DocC structure, and where it made absolute sense to break up paragraphs.

Reply:
👍 that works for me.